app/controllers/auth_key_pair_cloud_controller.rb

```diff
@@ -161,7 +161,7 @@ def delete_auth_key_pairs
     end
 
     if key_pairs.empty?
-      add_flash(_("No #{ui_lookup(:tables => 'auth_key_pair_cloud')} were selected for deletion"), :error)
+      add_flash(_("No %#{models} were selected for deletion") % {:models =>  ui_lookup(:tables => 'auth_key_pair_cloud')}, :error)
```

Comment:
`%#{models}` is incorrect. Besides, I'd much rather prefer to have the string expanded like:
`"No Key Pairs were selected for deletion"`